task22/js/main.js

```diff
@@ -128,21 +123,33 @@ const changeSortStatus = (status) => {
     }
 }
 
+const addEventForSortButton = (userContentsData) => {
+    const sortButtons = document.querySelectorAll(".js-sortButton");
+    sortButtons.forEach((button) => {
+        button.addEventListener('click', (e) => {
+            const nextStatus = changeSortStatus(e.target.dataset.status);
+            e.target.dataset.status = nextStatus;
+            button = e.currentTarget.id;
+            sortContents(button, userContentsData, nextStatus);        
+        });
+    });
+}
+
 const renderTableData = (userContentsData) => {
     document.getElementById("js-tableBody").remove();
     const table = document.querySelector("table");
     table.appendChild(getCreatedTableBody(userContentsData));
 };
 
-const sortContents = (userContentsData, status) => {
+const sortContents = (button, userContentsData,status) => {
```

Comment:
Same as above, but I think 'button' is confusing. I feel that targetColumn, etc. would be better.


```suggestion
const sortContents = (targetColumn, userContentsData,status) => {
```

Reply:
Same here.
[5689b91](https://github.com/seeeen044/JavaScript_handson/pull/24/commits/5689b91fc27f280f719cf08ea7c31de4ac40dd7e)